config/urls.py

```diff
@@ -14,62 +13,62 @@
 from metaci.repository.views import github_webhook
 
 urlpatterns = [
-    url(
+    re_path(
         r"^robots\.txt$",
         TemplateView.as_view(template_name="robots.txt", content_type="text/plain"),
     ),
-    url(r"^$", build_views.build_list, name="home"),
-    url(
+    re_path(r"^$", build_views.build_list, name="home"),
+    re_path(
         r"^about/$",
         mbci_views.AboutView.as_view(template_name="pages/about.html"),
         name="about",
     ),
-    # Django Admin, use {% url 'admin:index' %}
-    url(settings.ADMIN_URL_ROUTE, admin.site.urls),
+    # Django Admin, use {%  re_path 'admin:index' %}
```

Comment:
```suggestion
    # Django Admin, use {%  url 'admin:index' %}
```

I think this templatetag example was accidentally included in your search-and-replace